### PR TITLE
feat: moved Base aspects from DB UI Core to DB UI Base

### PR DESCRIPTION
**Pull request by nmerget**
partly solves https://github.com/db-ui/core/issues/129

### CODE REVIEW

File: scripts/color-classes-generator.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const generateBGVariants = (value, index) => {
  * @returns scss string
  */
 exports.generateColorUtilitityClasses = (colorToken) => {
-	let output = '@import "variables";\n@import "color-placeholder";\n';
+	let output = '@use "variables" as *;\n@use "color-placeholder" as *;\n';
```
Comment (mfranzke) [inline scripts/color-classes-generator.js:38]: template literals might be easier to read here.

File: scripts/scss-typography-generator.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const prefix = 'db';
 
 const fileHeader =
-	'@import "icon-family-calc";\n' +
+	'@use "variables" as *;\n@use "icon/icon-family-calc" as *;\n' +
```
Comment (mfranzke) [inline scripts/scss-typography-generator.js:4]: template literals might be easier to read here and at line 115

File: scss/_default.assets-paths.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$icons-path: "assets/icons/";
+$images-path: "assets/images/";
+$fonts-path: "assets/fonts/";
```
Comment (mfranzke) [inline scss/_default.assets-paths.scss:3]: it might be beneficial to make these themeable by adding a !default at the end (before the semicolon).

**Review by mfranzke — APPROVED** [review]

File: scss/_tonality.scss [hunk 4]
```diff
@@ -21,15 +22,15 @@
 		.db-ui-#{nth($styles, $i + 1)} {
 			@extend %db-scaling-#{nth($styles, $i + 1)};
 
-			[data-size="large"] {
-				@extend %db-#{nth($styles, $i + 1)}-body-lg;
-			}
-
-			:not(small),
+			*,
```
Comment (mfranzke) [inline scss/_tonality.scss:25]: CSS Variables, and the defined `font-size` and `line-height` are inherited, so I'm wondering whether we write it like this instead:
```SCSS
			// General styling
			@extend %db-#{nth($styles, $i + 1)}-body-md;
			
			[data-size="medium"] {
				@extend %db-#{nth($styles, $i + 1)}-body-md;
			}
```

File: scripts/color-classes-generator.js (hunk 1 above)
Comment (nmerget) [reply scripts/color-classes-generator.js]: fixed

File: scss/_default.assets-paths.scss (hunk 3 above)
Comment (nmerget) [reply scss/_default.assets-paths.scss]: fixed

File: scripts/scss-typography-generator.js (hunk 2 above)
Comment (nmerget) [reply scripts/scss-typography-generator.js]: fixed

File: scss/_tonality.scss (hunk 4 above)
Comment (nmerget) [reply scss/_tonality.scss:25]: This is not working...
I had this before, have you tried this?

**Pull request merged**